engine/crates/engine-v2/config/src/lib.rs

```diff
@@ -29,6 +30,8 @@ pub enum VersionedConfig {
     V2(v2::Config),
     /// V3 is like V2 but with FederatedGraphV2
     V3(v3::Config),
+    /// V4 is like V3 but with disable_introspection
```

Comment:
You don't need to add a `V4` for that, it's not a breaking change. Just add a `#[serde(default)]` or `#[serde(default = "func")]` and you're good ;)

Reply:
Done